### ISSUE TITLE
`window.titleBarStyle` should be `native` by default for SR users

### ISSUE THREAD

**Issue by meganrogge** [microsoft/vscode#204862]
@jooyoungseo has said that the native style is more accessible

**Comment by bpasero** [comment]
I am not sure I would agree here: the native menu on Windows is not accessible, and that is actually why we initially implemented our own custom menu that checks of the accessible requirements. So if we go down this path we have to really make sure that the native menu on Windows is now accessible (it is possible that Electron improved it meanwhile).

//cc @deepak1556

**Comment by deepak1556** [comment]
Electron fixed the accessibility of menus on Windows via https://github.com/electron/electron/pull/15302 which landed first in v5, @meganrogge please test and let me know if the native menu is not accessible with latest stable.

**Comment by meganrogge** [comment]
I don't have access to windows atm, but was told by @jooyoungseo that the native menu is very accessible on windows

**Comment by jooyoungseo** [comment]
Yes, the context menu (right-click popup) adds accelerator key at the end of each item so that keyboard-only users can quickly press that single-letter key to jump to the desired item. The Electron title bar and its context menu do not offer these accelerator keys for some reason.

**Comment by benibenj** [comment]
Is there a way to tell if it is a SR user on startup? I would need to know by the time this runs:
https://github.com/microsoft/vscode/blob/68410e1431f9849547b67d0b38dbc76cdb4aa3c8/src/vs/workbench/electron-sandbox/desktop.contribution.ts#L233-L239

I fear that the first time we launch vscode, we will not know if it is a SR user and show the custom title bar and then the second time we launch vscode we might know and suddenly show the native title bar. This could be confusing to the user

**Comment by meganrogge** [comment]
Checking the value of `editor.accessibilitySupport` would indicate that. There's also the `accessibilityService.isScreenReaderOptimized()` check.
